src/Workspaces/Remote/ServiceHub/Services/SourceGeneration/RemoteSourceGenerationService.cs

```diff
@@ -64,11 +71,65 @@ protected override IRemoteSourceGenerationService CreateService(in ServiceConstr
         }, cancellationToken);
     }
 
-    public ValueTask<bool> HasGeneratorsAsync(Checksum solutionChecksum, ProjectId projectId, CancellationToken cancellationToken)
+    private static readonly Dictionary<string, (AnalyzerReferenceMap analyzerReferenceMap, AnalyzerReferenceMap.CreateValueCallback callback)> s_languageToAnalyzerReferenceMap = new()
```

Comment:
>s_languageToAnalyzerReferenceMap [](http://example.com/codeflow?start=143&length=32)

nit (style preference): move to top of file

Reply:
will do (though i would like to unblock this pr).  so nit changes will come in followup.